src/main/java/br/com/codenation/v1/errorManager/user/User.java

```diff
@@ -1,28 +1,21 @@
 package br.com.codenation.v1.errorManager.user;
 
+import br.com.codenation.v1.errorManager.application.Application;
 import br.com.codenation.v1.errorManager.enums.Profile;
 import com.fasterxml.jackson.annotation.JsonFormat;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import org.springframework.data.annotation.CreatedDate;
 import org.springframework.data.annotation.LastModifiedDate;
 import org.springframework.data.jpa.domain.support.AuditingEntityListener;
 
-import javax.persistence.CollectionTable;
-import javax.persistence.Column;
-import javax.persistence.ElementCollection;
-import javax.persistence.Entity;
-import javax.persistence.EntityListeners;
-import javax.persistence.FetchType;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
-import javax.persistence.Table;
+import javax.persistence.*;
```

Comment:
Só pra não perder track do que estamos importando, acho que vale evitar importar com * e deixar como estava...
Mas é só uma sugestão mesmo. Tem que configurar a IDE, se estiver usando IntelliJ:
Go to Preferences (⌘ + , on macOS / Ctrl + Alt + S on Windows and Linux) > Editor > Code Style > Java > Imports tab set Class count to use import with '*' and Names count to use static import with '*' to a higher value. Any value over 99 seems to work fine.